phoenix-core/src/it/java/org/apache/phoenix/query/MetaDataCachingIT.java

```diff
@@ -152,12 +156,19 @@ public void testGlobalClientCacheMetrics() throws Exception {
         simulateWorkload("testGlobalClientCacheMetrics", numTables, numThreads, numMaxDML);
 
         // only 1 miss when the table is created
+        int numExpectedMisses = 1;
+        if (QueryServicesOptions.DEFAULT_LAST_DDL_TIMESTAMP_VALIDATION_ENABLED) {
```

Comment:
Why not to get the last ddl timestamp validation enabled flag from the connection properties?

Reply:
Not sure if there is a connection here in the test to use?

Reply:
@shahrs87 I have made the change in MetaDataClient.java as that is production code. For the test code, maybe it is not required? Also in some tests, we do not have a connection object which can be used directly for `getValidateLastDdlTimestampEnabled`

Reply:
I will make the change in tests to get the value from the config object in BaseTest.